### DISCUSSION TITLE
Will anything work without Home Hub?

### DISCUSSION THREAD

**Discussion by linusprojects** [HomeSpan/HomeSpan#870] [Q&A]
I have a few devices connected to my Apple Home app. Will I be able to control anything through HomeSpan if I don't have a Home Hub?

**Comment by HomeSpan** [comment]
HomeSpan provides the same functionality as a commercial device, so you will be able to use without a Home Hub, but with less functionality than with a Home Hub (as described on the HomeSpan main page).

**Reply by linusprojects** [reply]
Thank you for the quick response :)
